test/functional/android/multi_action_tests.py

```diff
@@ -1,172 +0,0 @@
-#!/usr/bin/env python
-
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-import unittest
-from time import sleep
-
-from appium import webdriver
-from appium.webdriver.common.touch_action import TouchAction
-from appium.webdriver.common.multi_action import MultiAction
-import desired_capabilities
-
-# the emulator is sometimes slow and needs time to think
-SLEEPY_TIME = 1
-
-
-class MultiActionTests(unittest.TestCase):
-    def setUp(self):
-        desired_caps = desired_capabilities.get_desired_capabilities('ApiDemos-debug.apk')
-        self.driver = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)
-
-    def tearDown(self):
-        self.driver.quit()
-
-    def test_parallel_actions(self):
```

Comment:
Why did you get rid of entire this test, not only `test_driver_pinch` and `test_driver_zoom`?
(Did not work?)

Reply:
Ah! I looked at the ios multitouch file and assumed the android one was a duplicate. Thanks for catching! I'll fix it.

Reply:
Fixed. Just deleting the two tests now.